modular_skyrat/modules/organs/lungs.dm

```diff
@@ -77,9 +77,9 @@
 	icon = 'modular_skyrat/modules/organs/lungs.dmi'
 	desc = "A set of lungs adapted to lower-pressure environments, though more susceptible to extreme temperatures."
 	icon_state = "lungs_toxin"
-	safe_oxygen_min = 5
+	safe_oxygen_min = 2
 
-	hot_message = "You can't stand the searing heat with every breath you take!"
+	hot_message = "You can't stand the searing heat with every breath you take"
```

Comment:
```suggestion
	hot_message = "You can't stand the searing heat with every breath you take!"
```
Honestly, at this point, this should be a define, or you should do a subset of lungs for your adapted lungs so they can share the same default message and not have to copy-paste it for every single one of them.

Reply:
Its on purpose :)